### ISSUE TITLE
Could not parse DOM element properties

### ISSUE THREAD

**Issue by tangshuang** [peerigon/angular-expressions#21]
In my case, I give a DOM element to scope, and want to parse out the value property, but not work:

```js
const el = document.querySelector('#input')
const ev = expressions.compile('el.value')
console.log(ev({ el }))
```

I get undefined, could you please to fix this?

**Comment by edi9999** [comment]
Hello @tangshuang , I don't have the time to put active work like this on the project because of other priorities.
If you write a pull request, I will look at it.

**Comment by tangshuang** [comment]
@cdelaorden you are right, I modified the code, see here https://github.com/tangshuang/scopex/blob/master/index.js
